src/js/components/container/Canvas.js

```diff
@@ -61,19 +62,29 @@ export default class Canvas extends React.Component {
           Math.round(clientX - xOffset + 1),
           Math.round(clientY - yOffset + 1)
         ];
-
+        if (coords[0] < 10 || coords[0] > 490 || coords[1] < 10 || coords[1] > 490){
```

Comment:
Can you store these values in variables that depend on the canvas size? This way they'll be more adaptive as we change the width and height of the canvas over time.